node/commons/compatibility-layer.ts

```diff
@@ -254,8 +253,7 @@ const fillSearchItemWithSimulation = (searchItem: SearchItem, orderFormItems: Or
       }
       const unitMultiplier = orderFormItem.unitMultiplier ? orderFormItem.unitMultiplier : 1
       const { listPrice, price, priceValidUntil, sellingPrice } = orderFormItem
-
-      seller.commertialOffer.AvailableQuantity = orderFormItem?.availability === 'available' ? 10000 : 0
+      seller.commertialOffer.AvailableQuantity = orderFormItem.stockBalance && orderFormItem.stockBalance > '0' && orderFormItem?.availability === 'available' ? 10000 : 0
```

Comment:
```suggestion
      seller.commertialOffer.AvailableQuantity = orderFormItem.stockBalance && orderFormItem?.availability === 'available' ? 10000 : 0
```
`stockBalance` is a number, it will already be falsy if the value is zero